src/model/model_json.nit

```diff
@@ -166,6 +166,7 @@ end
 redef class MClassDef
 	redef fun json do
 		var obj = super
+		obj["mdoc"] = mdoc
```

Comment:
Since Nitweb relies on the JSON model, I won’t touch the current behaviour without the consent of @Morriar.